src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.targets.in

```diff
@@ -54,4 +73,43 @@
         <Import Project="Sdk.targets" Sdk="Microsoft.NET.Runtime.MonoTargets.Sdk" />
     </ImportGroup>
 
+    <PropertyGroup Condition="'$(TargetPlatformIdentifier)' == 'android' or '$(TargetPlatformIdentifier)' == 'macOS' or '$(TargetPlatformIdentifier)' == 'ios' or '$(TargetPlatformIdentifier)' == 'maccatalyst' or '$(TargetPlatformIdentifier)' == 'tvos' or ('$(RuntimeIdentifier)' == 'browser-wasm' and '$(UsingBrowserRuntimeWorkload)' == 'true')">
+      <_MonoWorkloadTargetsMobile>true</_MonoWorkloadTargetsMobile>
+      <_MonoWorkloadRuntimePackPackageVersion>$(RuntimePackInWorkloadVersion)</_MonoWorkloadRuntimePackPackageVersion>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'$(_MonoWorkloadTargetsMobile)' == 'true'">
+      <MonoRuntimePackRids Include="
+        linux-x64;
+        win-x64;
+        osx-x64;
+        osx-arm64;
+        maccatalyst-x64;
+        maccatalyst-arm64;
+        browser-wasm;
+        ios-arm64;
+        ios-arm;
+        iossimulator-arm64;
+        iossimulator-x64;
+        iossimulator-x86;
+        tvos-arm64;
+        tvossimulator-arm64;
+        tvossimulator-x64;
+        android-arm64;
+        android-arm;
+        android-x64;
+        android-x86;
+        " />
+      
+      <KnownRuntimePack Remove="Microsoft.NETCore.App" />
+      <KnownRuntimePack Include="Microsoft.NETCore.App"
+                        TargetFramework="net6.0"
```

Comment:
@lewing @steveisok while this works for now, it'll break once we move to `net7.0`. Can you try whether doing this instead works:

```xml
<KnownRuntimePack Update="Microsoft.NETCore.App"
                  LatestRuntimeFrameworkVersion="$(_MonoWorkloadRuntimePackPackageVersion)"
                  Condition="'%(RuntimePackLabels)' == 'Mono' and '%(TargetFramework)' == 'net$(TargetFrameworkVersion)'"
```

Reply:
Sure thing